interface/VideoCommonDefs.h

```diff
@@ -25,6 +25,10 @@
 
 #include <stdint.h>
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
```

Comment:
It is generally not a good idea to include config.h in headers.  There's a comment about this 4 lines above.